roles/openshift_excluder/tasks/install.yml

```diff
@@ -2,15 +2,15 @@
 - name: Install latest excluder
   package:
     name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
+    state: "{{ excluder_package_state }}"
   when:
   - openshift_excluder_enabled | default(false) | bool
   - not openshift.common.is_containerized | bool
 
 - name: Install latest docker excluder
   package:
     name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
+    state: "{{ docker_excluder_package_state }}"
   when:
   - docker_excluder_enabled | default(false) | bool
   - not openshift.common.is_containerized | bool
```

Comment:
The problem is not the ``openshift_excluder`` role itself but playbooks that use the role. Both install and upgrade are included by common playbook that is responsible for unexcluding excluders. Part of the unexclude process is running ``openshift_excluder.status`` which installs both excluder rpms. I will need to break the common playbook into two and run install/upgrade task accordingly.